run.sh

```diff
@@ -106,6 +106,8 @@ service cups restart
 echo "export EYEOS_UNIX_USER=$EYEOS_UNIX_USER" > /tmp/global.env
 echo "export BUS_ADDRESS_HOST=$BUS_ADDRESS_HOST" >> /tmp/global.env
 echo "export BUS_SUBSCRIPTION=$BUS_SUBSCRIPTION" >> /tmp/global.env
+echo "export EYEOS_BUS_MASTER_PASSWD=$EYEOS_BUS_MASTER_PASSWD" >> /tmp/global.env
```

Comment:
yes, it is the user card. I'd like someone to fix this before it gets merged because it is wildly misleading.